.idea/src/lesson17/Task17/CollEctions.java

```diff
@@ -0,0 +1,76 @@
+package lesson17.Task17;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+class CollEctions {
+    public static void main(String[] args) {
```

Comment:
Why do you need main method here? The task was to have a Collection class with field of `Number[]`